index.html

```diff
@@ -3138,9 +3138,95 @@ <h2>DID Resolution</h2>
 consumption rules.">
         <figcaption>
 Functions resolve() and resolveRepresentation().
+See also: <a class="longdesc-link"
+href="#resolve-resolverepresentation-longdesc">narrative description</a>.
         </figcaption>
       </figure>
 
+        <div class="longdesc" id="resolve-resolverepresentation-longdesc">
+            <p>
+                The upper middle part of the diagram contains a rectangle with dashed grey outline, containing two
+                blue-outlined rectangles, one above the other, as follows.
+                The upper, larger rectangle is labeled, in blue, "Core Properties", and contains the following [INFRA]
```

Comment:
```suggestion
                The upper, larger rectangle is labeled, in blue, "Core Properties", and contains the following [INFRA]
                The upper, larger rectangle is labeled, in blue, "Core Properties", and contains the following <a data-cite="INFRA#maps">INFRA</a>
```
It seems double brackets (`[[INFRA]]`) are needed to make it a link. But in #781 I linked directly to the "maps" section instead, since that is where the notation is explained in the INFRA specification (`A literal syntax can be used to express ordered maps, by surrounding the ordered map with «[ ]» characters, denoting each of its entries as key → value, and separating its entries with a comma.`)

Reply:
Fixed in https://github.com/w3c/did-core/pull/785/commits/8c3b32f6b6876d28ba2e4a2463153383cb9f0b37.